packages/cli-kit/src/haiku.ts

```diff
@@ -1,9 +1,77 @@
-import Haikunator from 'haikunator'
+import {file, path} from './index'
 
-const haikunator = new Haikunator()
+export const SAFE_ADJECTIVES = [
+  'commercial',
+  'profitable',
+  'amortizable',
+  'branded',
+  'integrated',
+  'synergistic',
+  'consolidated',
+  'diversified',
+  'lean',
+  'niche',
+  'premium',
+  'luxury',
+  'scalable',
+  'optimized',
+  'empowered',
+  'international',
+  'beneficial',
+  'fruitful',
+  'extensive',
+  'lucrative',
+  'modern',
+]
 
-export function generate(suffix: string): string {
-  const generated = haikunator.haikunate()
-  const [adjective, noun, token] = generated.split('-')
-  return [adjective, noun, suffix, token].join('-')
+export const SAFE_NOUNS = [
+  'account',
+  'consumer',
+  'customer',
+  'enterprise',
+  'business',
+  'venture',
+  'marketplace',
+  'revenue',
+  'vertical',
+  'portfolio',
+  'negotiation',
+  'shipping',
+  'demand',
+  'supply',
+  'growth',
+  'merchant',
+  'investment',
+  'shareholder',
+  'conversion',
+  'capital',
+  'projection',
+  'upside',
+  'trade',
+  'deal',
+  'merchandise',
+  'transaction',
+  'sale',
+]
+
+export async function generate({
+  suffix,
+  directory,
+  randomPick = (array: string[]) => array[Math.floor(Math.random() * array.length)],
```

Comment:
I just wanted to make the function injectable but I can definitely add a constant so that this becomes shorter.

Reply:
If this is only for testing, it can be it's own function and it can be mocked.

A `randomPick` function might be needed in another places, I think we should make it reusable (and you can mock it if needed for tests!)